charts/tfy-agent/values.yaml

```diff
@@ -432,6 +432,66 @@ tfyAgentProxy:
     ## 
     name: ""
 
+  clusterRole:
+    ## @param tfyAgentProxy.clusterRole.enable Create cluster role.
+    enable: true
+    ## @param tfyAgentProxy.clusterRole.rules [array] Cluster role Authz rules.
+    rules:
+      - apiGroups: [""]
+        resources: ["namespaces", "resourcequotas"]
+        verbs: ["get", "delete", "patch", "list", "create"]
+
+      - apiGroups: [""]
+        resources: ["resourcequotas/status"]
+        verbs: ["get"]
+
+      - apiGroups: [""]
+        resources: ["secrets"]
+        verbs: ["create", "patch"]
+
+      - apiGroups: [""]
+        resources: ["pods"]
+        verbs: ["delete"]
+
+      - apiGroups: [""]
+        resources: ["pods/log", "events", "nodes", "services", "persistentvolumes"]
+        verbs: ["get", "list"]
+
+      - apiGroups: ["storage.k8s.io"]
+        resources: ["storageclasses"]
+        verbs: ["get", "list"]
+
+      - apiGroups: ["apps"]
+        resources: ["statefulsets/scale"]
+        verbs: ["patch"]
+
+      - apiGroups: ["networking.istio.io"]
+        resources: ["virtualservices"]
+        verbs: ["list"]
+
+      - apiGroups: ["keda.sh"]
+        resources: ["scaledobjects"]
+        verbs: ["get", "update"]
```

Comment:
Should we also have `list` wherever we have `get`?

Reply:
list allows you to discover new resources. get does not, you need to know the name.